README.md

```diff
@@ -4,24 +4,35 @@
 - Here is also a good [Docker tutorial](https://www.youtube.com/watch?v=pg19Z8LL06w) on YouTube.
 
 ## Build
-Build an image from the Dockerfile in the current directory and tag the image, e.g.:
-> `docker build -f Dockerfile -t zaemyung/ml:latest .`
+Download the [Dockerfile](https://github.com/zaemyung/dockerfiles/blob/39d3ba38d37ce3f4ce8ab4a126cc4a71e105f8f9/ml/Dockerfile) from `zaemyung/ml` to the current directory. 
```

Comment:
This will point to specific git commit. Maybe this is better? `https://github.com/zaemyung/dockerfiles/blob/master/ml/Dockerfile`